src/index.html

```diff
@@ -252,12 +252,13 @@
                 src="assets/dash/2x/3-1 - aot compile@2x-80.jpg"/>
         </div>
     </section>
-    <section class="row dash-row">
+    <section class="row">
         <div class="dash-dartpad">
-            <h2>Try Dart in your browser!</h2>
+            <a name="try-dart"></a>
+            <h2>Try Dart in your browser</h2>
             <select id="dartpad-select"></select>
             <div id="dartpad-host"></div>
-            <h3>Another call to action</h3>
+            <h3>Want more practice? <a href="/codelabs">Try a codelab.</a> Or <a href="/get-dart">download Dart</a>.</h3>
```

Comment:
</a>. -> .</a>

(For consistency. I don't usually like having the punctuation within the link text, but it looks better to me here.)